addons/web_editor/static/src/js/wysiwyg_snippets/wysiwyg_snippets.js

```diff
@@ -71,17 +71,11 @@ Wysiwyg.include({
                 $editable.before($wrap);
                 $wrap.append($editable);
                 setTimeout(function () { // add a set timeout for the transition
-                    var totalDisabledSnippets = 0;
-                    _.each(self.snippets.$snippets, function (el) {
-                        if ($(el).hasClass('o_disabled')) {
-                            totalDisabledSnippets += 1;
-                        }
-                    });
-                    if (self.snippets.$snippets.length !== totalDisabledSnippets) {
+                    if (self.snippets.$snippets.not('.o_disabled').length > 0) {
```

Comment:
@qsm-odoo What do you think of this ? If all snippets are disabled, it just does not add `o_loaded` and `o_snippets_loaded` css classes so the snippets bar stays hidden. This said, by taking a closer look I don't really understand the use of `self.trigger_up('snippets_loaded', self.snippets.$el);`, is this even called ?

Reply:
Grep 'snippets_loaded' and you will have your answer

Reply:
This would be cheeky not to grep it before asking :smile: Sure I did it and took a look a the associated custom_event but I did not really understand  what is the use.. but anyway it does not make much difference, I'll take a deeper look at your explanation.

Reply:
Apparently, it is to make a customization in mass mailing. Loading the mailing themes once the snippets are loaded and apparently leaving console.log in the code... but well, new editor :sneezing_face: